### ISSUE TITLE
[package]  capnproto/0.8.0: builddirs should be a list

### ISSUE THREAD

**Issue by lasote** [conan-io/conan-center-index#5438]
The recipe declares builddirs like this: 

```
self.cpp_info.components["kj"].builddirs = self._cmake_folder
```

But it should be a list:
```
self.cpp_info.components["kj"].builddirs = [self._cmake_folder]
```

It is causing the generators to provide wrong values.

### Package and Environment Details (include every applicable attribute)
  * Package Name/Version: **capnproto/0.8.0**
  * Operating System+version: **OSX**

**Comment by uilianries** [comment]
Could we validate it in Conan client? For hooks we could use post_package_info, but it sounds be a built-in feature for Conan.

**Comment by SpaceIm** [comment]
Yes, forgotten to update capnproto. Actually it should even append to builddirs, there might be side effects when package folder is removed from builddirs.
